cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1893,34 +1893,34 @@ impl_runtime_apis! {
 						xcm_config::XcmConfig,
 						ExistentialDepositAsset,
 						PriceForSiblingParachainDelivery,
-						RandomParaId,
+						SiblingAssetHubId,
 						ParachainSystem,
 					>
 				);
 
 				fn reachable_dest() -> Option<Location> {
-					Some(Parent.into())
+					Some(AssetHubWestend::get())
 				}
 
 				fn teleportable_asset_and_dest() -> Option<(Asset, Location)> {
 					// Relay/native token can be teleported between AH and Relay.
 					Some((
 						Asset {
 							fun: Fungible(ExistentialDeposit::get()),
-							id: AssetId(Parent.into())
+							id: AssetId(AssetHubWestend::get())
```

Comment:
watch out, here `Parent.into()` is used in the context of AssetId, so change it better tot `WestendLocation`
```suggestion
							id: AssetId(WestendLocation::get())
```